airflow/cli/commands/internal_api_command.py

```diff
@@ -102,7 +102,7 @@ def internal_api(args):
             "--workers",
             str(num_workers),
             "--worker-class",
-            str(args.workerclass),
+            "uvicorn.workers.UvicornWorker",
```

Comment:
TODO: We need to remove option of using different workers and only leave the uvicorn worker.